.ghc.environment.x86_64-linux-8.6.5

```diff
@@ -0,0 +1,104 @@
+package-id ansi-terminal-0.11.1-4afb52e883c7d1d9279d75ac44fb6ccb7d51be920527fddcb3ccad0ea54a2410
+package-id colour-2.3.5-7cc4ff67f62b39f31668a50554a26aea929c746c154f83be89c7eaed763b8428
+package-id filepath-1.4.2.1
+package-id gitrev-1.3.1-f67a19b9613919d81229ea302cc0f31da4720fe75f308b8e311ddb7bcaac3062
+package-id directory-1.3.3.0
+package-id process-1.6.5.0
+package-id path-0.8.0-e64e9d95e66b491b033466816387ae5c2100059028c7626cc799b158f8aa8643
+package-id exceptions-0.10.4-2e686fdff7a6bdbd62e24641169087c094e991de8ab3f6381a859306fd258e32
+package-id typed-process-0.2.8.0-9b1b906245fe30f02838f795c8067141808a81ad5d14a0e130a6567addfb1eb4
+package-id async-2.2.4-eaf6ead38a715a9a80a863a19ae4eb6b02adcefb56778cec4a968f62dd6f81bc
+package-id unliftio-core-0.2.0.1-2508a890d12eafb8f47f24cabe44202bd7d562a64815930cd36681e67e86553b
+package-id yaml-0.11.7.0-40fc9b444d2d706124da5db72d4728b105f3fa72878f73369049cc4dabecc076
+package-id conduit-1.3.4.2-1f8ca4ce5f9f64ca445613771abe7e2dff4db4ccfc9c0e27a0379ac11d12bc41
+package-id mono-traversable-1.0.15.1-ca45559ebaa9a3efe6fd2d236ba58971b67b0599f2506ee8bc895b68ace4979b
+package-id split-0.2.3.4-c3f2b1332346c55ce7f7ea363228ab95893722bb9cbd3d503e93c848b358115a
+package-id vector-algorithms-0.8.0.4-e9e58f4ce6e97c82ac4fe5dd73a63c63e97b9efadac9e42d67dfee6b2025f111
+package-id resourcet-1.2.4.3-816cab9802b090e1f0b2a94317bec3b658dfcda698166fcb1b425d67e015648a
+package-id libyaml-0.1.2-460a36b04da6a465d2922ff5461b3ce0cf94b0c1f79d669c216692035274cbef
+package-id clock-0.8.3-2c0ba435833a4e559a124209e85aec1fdc08a9b6eb74fbd1211b59493c5c10a9
+package-id path-io-1.6.2-34905c7edc1492cfdab059055efa522c270df0e7330623c0b498aa91cf54da20
+package-id temporary-1.3-081f227532e200d9c35a176e557bc61abdf4c23405f989b81340c16696919078
+package-id unix-compat-0.5.4-114d067b4b3aa5f24508e3ed1550cd5bb25ee21e2f81a2219d6cae8ff2e5e389
+package-id hspec-2.9.4-929bdada373072968d23b792d0f8e3cdeac28b90ed5d0ef4f4ecc51638cd2287
+package-id hspec-core-2.9.4-9e99973af2f0c894247808485907c547f7541ec230baec8130fb353170aa88b9
+package-id HUnit-1.6.2.0-e4caf450ed1503d3aee02d6737fc9342e978586be5ed6877dd21ba75ff15b254
+package-id call-stack-0.4.0-69a657f89f4fd643daf1369d8e7a50facb59497e2b6d1a8434e19f3d88b6e6bd
+package-id ghc-8.6.5
+package-id ghc-boot-8.6.5
+package-id ghc-heap-8.6.5
+package-id ghci-8.6.5
+package-id hpc-0.6.0.3
+package-id terminfo-0.4.1.2
+package-id hspec-expectations-0.8.2-bad364504a87c5d25afa8883e15da1534b995c709f56a6780b5ee8050114853d
+package-id quickcheck-io-0.2.0-51fb987a81cb57dc8d982802798e3f4db159e0c11c34a86ac43ba98be11ebfbc
+package-id setenv-0.1.1.3-70d0339fd3bf6d61fc1b27eae561267df28e95a9ce8f965eb983a1bcac0b6bf7
+package-id tf-random-0.5-c08496a58a6b5a983648829ed712c43605d63581981351c4d5fb12e23c1d3a06
+package-id hspec-discover-2.9.4-710a59499685d42180b0922b252e9695ab41f5e51b319b5b16ff075e3419778a
+package-id silently-1.2.5.2-18795575b96093d3652fc030d5b61bf2e75aa6e364ac18918cce4ad5a803bb76
```

Comment:
Hey @FranzGB Maybe this file is related to your environment. Can you please remove it?

Reply:
Sure, done :)